src/app/api/crud/groups.py

```diff
@@ -29,31 +30,25 @@ async def get_entity_group_id(table: Table, entry_id: int) -> int:
     elif table == alerts:
         return await _get_alert_group_id(entry_id)
 
-    raise f"{table} is not an existing entity"
+    raise ValueError(f"Unknown table '{table.name}'")
 
 
 async def _get_access_group_id(entry_id: int) -> int:
     entry = await crud.base.get_entry(accesses, entry_id)
-    if entry is None:
-        return None
     return entry["group_id"]
 
 
 async def _get_user_group_id(entry_id: int) -> int:
     entry = await crud.base.get_entry(users, entry_id)
-    if entry is None:
-        return None
     return await _get_access_group_id(entry["access_id"])
 
 
 async def _get_site_group_id(entry_id: int) -> int:
     entry = await crud.base.get_entry(sites, entry_id)
-    if entry is None:
```

Comment:
won't it generate errors if entry is none andn we try to call entry["ghroup_id"]?

Reply:
the `crud.base.get_entry` cannot return a `None`. If it founds a `None` entry, the function itself throws a 404 error, so I remove the case that won't ever happen :)